### PR TITLE
[pipelined] Fix Tunnel ID Proto Field Type 

### PR DESCRIPTION
**Pull request by koolzz**
Signed-off-by: Nick Yurchenko <koolzz@fb.com>

## Summary

I'm not sure why I initially thought the tunnels would be string based for OVS, this pr fixes that

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/protos/session_manager.proto [hunk 1]
```diff
@@ -103,8 +103,9 @@ message LocalEndSessionResponse {
 message UpdateTunnelIdsRequest {
   SubscriberID sid = 1;
   uint32 bearer_id = 2;
-  string enb_teid = 3;
-  string agw_teid = 4;
+  uint32 enb_teid = 5;
+  uint32 agw_teid = 6;
+  reserved 3,4;
```
Comment (ulaskozat) [inline lte/protos/session_manager.proto]: is there a reason why we do not simply use 3 and 4 for these fields, there is no legacy for this feature.

File: lte/protos/pipelined.proto [hunk 2]
```diff
@@ -77,8 +77,9 @@ message ActivateFlowsRequest {
   magma.lte.AggregatedMaximumBitrate apn_ambr = 6; // APN AMBR info, passed at the time of session creation
   bytes ipv6_addr = 7; // Subscriber session ipv6 address
   bytes msisdn = 8; // From SubscriberDB or HSS
-  string uplink_tunnel = 9; // Subscriber uplink tunnel
-  string downlink_tunnel = 10; // Subscriber downlink tunnel
+  uint32 uplink_tunnel = 11; // Subscriber uplink tunnel
+  uint32 downlink_tunnel = 12; // Subscriber downlink tunnel
+  reserved 9,10;
```
Comment (ulaskozat) [inline lte/protos/pipelined.proto]: is there a reason why we don't reuse 9 and 10?

**Review by ulaskozat — APPROVED** [review]
LGTM with nit comments

File: lte/protos/session_manager.proto (hunk 1 above)
Comment (koolzz) [reply lte/protos/session_manager.proto]: Yeah changing type is unsafe from string to uint but I guess as we don't use this anywhere yet we can ignore that

File: lte/gateway/c/session_manager/test/test_sessiond_integ.cpp [hunk 3]
```diff
@@ -386,8 +386,8 @@ TEST_F(SessiondTest, end_to_end_success) {
   UpdateTunnelIdsRequest tun_update_request;
   tun_update_request.mutable_sid()->set_id("IMSI1");
   tun_update_request.set_bearer_id(default_bearer);
-  tun_update_request.set_enb_teid("10");
-  tun_update_request.set_agw_teid("20");
+  tun_update_request.set_enb_teid(10);
+  tun_update_request.set_agw_teid(20);
```
Comment (uri200) [inline lte/gateway/c/session_manager/test/test_sessiond_integ.cpp]: i think I forgot to use the vars here. Can you set this to 
```
  tun_update_request.set_enb_teid(enb_teid);
  tun_update_request.set_agw_teid(agw_teid);	
```

**Review by uri200 — APPROVED** [review]

**Review by themarwhal — APPROVED** [review]

**Review by pshelar — APPROVED** [review]
LGTM

**Pull request merged**